internal/dbtools/testing.go

```diff
@@ -0,0 +1,11 @@
+package dbtools
+
+import "github.com/cockroachdb/cockroach-go/v2/testserver"
+
+// TestServerCRDBVersion is the version of CockroachDB that the test server is running
+const TestServerCRDBVersion = "v24.3.8"
```

Comment:
do you think this should be latest or should it be the last CRDB with the BSL https://www.cockroachlabs.com/docs/releases/#licenses